Vostok.Logging.File/Configuration/RollingStrategyOptions.cs

```diff
@@ -27,5 +27,10 @@ public class RollingStrategyOptions
         /// Maximal size of one part of log file, in bytes. Affects only <see cref="RollingStrategyType.BySize"/> and <see cref="RollingStrategyType.Hybrid"/> strategies.
         /// </summary>
         public long MaxSize { get; set; } = 1024 * 1024 * 1024;
+
+        /// <summary>
+        /// Eliminator between base path and rolling suffix. Not affects <see cref="RollingStrategyType.None"/> strategy.
```

Comment:
"Not affects" --> "Does not affect"

Reply:
I also suggest adding a remark about `{RollingSuffix}` placeholder and its interaction with this setting (when does its value get inserted before the suffix?).